package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typy",
-  "version": "3.2.1",
+  "version": "3.2.2",
```

Comment:
We don't update package version as part of commits. Two reasons —

1. We use npm cli tool to automatically upgrade patch/minor/major versions. More about it [here](https://docs.npmjs.com/cli/version)
2. Sometimes PRs can be merged a bit later and other versions might have gone out before. So it's a good idea to leave the version management to maintainers and not include it as part of PRs in any project.

Reply:
Sorry for that! A bit over-excited! 😅 Will remove this change 👍